packages/contractkit/src/wrappers/Validators.test.ts

```diff
@@ -86,6 +86,16 @@ testWithGanache('Validators Wrapper', (web3) => {
     expect(members).toContain(validatorAccount)
   })
 
+  test('SBAT updateCommission', async () => {
+    const groupAccount = accounts[0]
+    await setupGroup(groupAccount)
+    await (await validators.updateCommission(new BigNumber(0.2))).sendAndWaitForReceipt({
+      from: groupAccount,
+    })
+    const commission = (await validators.getValidatorGroup(groupAccount)).commission
+    expect(commission.toFixed()).toEqual('0.2')
```

Comment:
there's a  `expect().toEqBigNumber` in contractKit that you can use (no need to import or anything)